addons/point_of_sale/static/src/app/screens/saver_screen/saver_screen.js

```diff
@@ -5,6 +5,7 @@ import { useTime } from "@point_of_sale/app/utils/time_hook";
 export class SaverScreen extends Component {
     static template = "point_of_sale.SaverScreen";
     static storeOnOrder = false;
+    static updatePreviousScreen = false;
```

Comment:
```suggestion
    static disablePreviousScreen = true;
```

Reply:
@davidmonnom The name disablePreviousScreen is misleading because it doesn’t actually disable the previous screen; it prevents the previous screen from being updated. Therefore, I named the flag to updatePreviousScreen and set its default value to true. This way, if updatePreviousScreen is false, it will not update the previous screen.

I considered using a negative flag name (with a default of false), but that would have resulted in an excessively long name. As long as the default remains true, this approach works well.

Reply:
The aim here is to prevent the component in question from being set to previousScreen